### DISCUSSION TITLE
Just another thankyou

### DISCUSSION THREAD

**Discussion by adenoz** [EdJoPaTo/mqttui#131] [General]
Hey, so this is excellent.

I've actually been considering building something like this. I've looked in the past and for some reason didn't find this. I randomly found this today browsing Github for various TUIs. Just got it working on macos no problem at all. Soooo much better than using MQTT Explorer. That has it's place of course, but I'll be using **mqttui** for quick checks and the like going forward.

I maintain https://github.com/athenars-io/rusqttbom and I'll be including a screenshot of the weather data coming in via mqttui in the README there. I am a big terminal fan and this find has been a highlight of my day.

When I get more time, I'm looking forward to going through the code and seeing what you've done.

So well done and thanks for making this!

**Comment by snova69** [comment]
wow! just found this too!   add me to the thank you list!! so cool.   mqtt explorer for Apple Silicon who?  wow.! thank you thank you.
